products/securityscanner/api.yaml

```diff
@@ -0,0 +1,157 @@
+                required: true
+                description: |
+                  The password of the Google account. The credential is stored encrypted
+                  and not returned in any response nor included in audit logs.
+          - !ruby/object:Api::Type::NestedObject
+            name: customAccount
+            description: |
+              Describes authentication configuration that uses a custom account.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: username
+                required: true
+                description: |
+                  The user name of the custom account.
+              - !ruby/object:Api::Type::String
+                name: password
+                input: true
+                required: true
+                description: |
+                  The password of the custom account. The credential is stored encrypted
+                  and not returned in any response nor included in audit logs.
+              - !ruby/object:Api::Type::String
+                name: loginUrl
+                required: true
+                description: |
+                  The login form URL of the website.
+      - !ruby/object:Api::Type::Enum
+        name: userAgent
+        description: Type of the user agents used for scanning
+        default_value: :CHROME_LINUX
+        values:
+          - :USER_AGENT_UNSPECIFIED
+          - :CHROME_LINUX
+          - :CHROME_ANDROID
+          - :SAFARI_IPHONE
+      - !ruby/object:Api::Type::Array
+        name: blacklistPatterns
+        description: |
+          The blacklist URL patterns as described in
+          https://cloud.google.com/security-scanner/docs/excluded-urls
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::NestedObject
+        name: schedule
+        description: |
+          The schedule of the ScanConfig
+        properties:
+          - !ruby/object:Api::Type::Time
+            name: scheduleTime
+            description: |
+              A timestamp indicates when the next run will be scheduled. The value is refreshed
+              by the server after each run. If unspecified, it will default to current server time,
+              which means the scan will be scheduled to start immediately.
+          - !ruby/object:Api::Type::Integer
+            name: intervalDurationDays
+            required: true
+            description: |
+              The duration of time between executions in days
+      - !ruby/object:Api::Type::Array
+        name: targetPlatforms
+        description: |
+          Set of Cloud Platforms targeted by the scan. If empty, APP_ENGINE will be used as a default.
+        item_type: !ruby/object:Api::Type::Enum
+          name: undefined
+          description: |
+            This field only has a name and description because of MM
+            limitations. It should not appear in downstreams.
+          values:
+            - :TARGET_PLATFORM_UNSPECIFIED
```

Comment:
is this a value that could realistically be set by someone? If not, I'd say just yank it out